### PR TITLE
[bitnami/harbor] Fix SCANNER_REDIS_URL for trivy-scanner

### PR DESCRIPTION
**Pull request by funkypenguin**
**Description of the change**

The latest harbor-trivy-scanner seems to require the env var `SCANNER_REDIS_URL`, whereas previous versions required `SCANNER_STORE_REDIS_URL` and `SCANNER_JOB_QUEUE_REDIS_URL` (although they were set to the same value)

This change updates the chart v8.x.x for the new env var, which is required to make Trivy work. In the absense of this change, the trivy pod will start, but fail to scan, outputting the following:

```
ERROR: worker.fetch - dial tcp [::1]:6379: connect: connection refused
ERROR: worker.fetch - dial tcp [::1]:6379: connect: connection refused
```

**Benefits**

Trivy works!

**Possible drawbacks**

None

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
`values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/harbor/templates/trivy/trivy-secret-envvars.yaml [hunk 1]
```diff
@@ -14,6 +14,5 @@ metadata:
 type: Opaque
 data:
   SCANNER_TRIVY_GITHUB_TOKEN: {{ .Values.trivy.gitHubToken | default "" | b64enc | quote }}
-  SCANNER_STORE_REDIS_URL: {{ include "harbor.redisForTrivyAdapter" . | b64enc }}
```
Comment (joancafom) [inline bitnami/harbor/templates/trivy/trivy-secret-envvars.yaml:17]: The upstream project seems to maintain the three env variables, which may be due to backwards compatibility with previous versions of the trivy container. What about keeping the three of them as well here to match this idea?

https://github.com/goharbor/harbor-helm/blob/f5767efaa35f835934ce060aa467b3a00af82837/templates/trivy/trivy-sts.yaml#L104

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for your PR! Could you please take a look at my comment?

File: bitnami/harbor/templates/trivy/trivy-secret-envvars.yaml (hunk 1 above)
Comment (funkypenguin) [reply bitnami/harbor/templates/trivy/trivy-secret-envvars.yaml:17]: Good idea, that seems safe(r) ;)

**Review by joancafom — APPROVED** [review]
LGTM! Thank you for your changes!

**Pull request merged**